src/views/Players/Player.test.js

```diff
@@ -0,0 +1,12 @@
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min';
+import Player from '../../views/Players/Player';
+
+test('renders title message', () => {
+  const container = render(
```

Comment:
Making this test async and awaiting with findByText would allow the Component's useEffect to run and render all of the data.